src/macos/StringPolicy.cc

```diff
@@ -0,0 +1,46 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+#include "StringPolicy.hh"
+
+using namespace Napi;
+
+StringPolicy::StringPolicy(const std::string name, const std::string &productName)
+    : PreferencesPolicy(name, productName)
+{
+}
+
+Value StringPolicy::getJSValue(Env env, std::string value) const
+{
+  return String::New(env, value);
+}
+
+std::optional<std::string> StringPolicy::read() const
+{
+  auto pref = CFPreferencesCopyAppValue(key, appID);
```

Comment:
There is also the [`CFPreferencesAppValueIsForced`](https://developer.apple.com/documentation/corefoundation/1515521-cfpreferencesappvalueisforced?language=objc) API that could let us be more restrictive on what we consider "policy" vs config that the user sets.  

Similar to https://github.com/MarshallOfSound/cf-prefs/tree/main?tab=readme-ov-file#permissionsispreferenceforcedkey